pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -82,13 +83,19 @@ func (pa *PodAutoscaler) annotationFloat64(key string) (float64, bool) {
 // `(min, max int32)`. The value of 0 for any of min or max means the bound is
 // not set.
 // Note: min will be ignored if the PA is not reachable
-func (pa *PodAutoscaler) ScaleBounds() (min, max int32) {
+func (pa *PodAutoscaler) ScaleBounds(asConfig *autoscalerconfig.Config) (int32, int32) {
+	var min, max int32
+
 	if pa.Spec.Reachability != ReachabilityUnreachable {
 		min, _ = pa.annotationInt32(autoscaling.MinScaleAnnotationKey)
 	}
-	max, _ = pa.annotationInt32(autoscaling.MaxScaleAnnotationKey)
 
-	return
+	max := asConfig.MaxScale
```

Comment:
this `:=` won't compile since max is already defined (that's why the tests are failing)

Reply:
🤦